kimchi/src/circuits/polynomials/range_check/gate.rs

```diff
@@ -113,6 +63,18 @@ impl<F: PrimeField> CircuitGate<F> {
             },
         ];
 
+        // copy v0p0
+        circuit_gates.connect_cell_pair((0, 1), (3, 3));
```

Comment:
I much prefer this! I think ideally we should really remove the wiring from `CircuitGate` and keep track of it separately, to make it safer to connect cells (by just keeping track of cycles)

Reply:
Thanks! In any case, I agree that there should be something separate for it. But at least this way is simpler, and it reuses the same function for both range check and ffadd (which was not the case initially)